boot/storage.txt

```diff
@@ -0,0 +1,285 @@
+    your disposal:
+
+    - gdisk
+    - parted
+    - fdisk
+    
+    
+[2.0] Partitioning Schemas
+________________________________________________________________________________
+
+Before creating a partition or multiple partitions, there are several things to
+take into consideration:
+
+    [2.1] Partition Number, Format and Size
+    ____________________________________________________________________________
+    
+    The number of partitions required is highly variable and dependent on the
+    environment.  The following table provides examples of common partition
+    schemas, formats and recommended sizes:
+    
+    +-------------------------------+------------------------------------------+
+    |    Schema Description         |   Partition (Type,      Format, Size)    |
+    +-------------------------------+------------------------------------------+
+    |                               |                                          |
+    |    GPT with UEFI Support      |   /dev/sda1 (BIOS BOOT, EXT4,   128MB)   |
+    |    with physical swap         |   /dev/sda2 (UEFI ESP,  FAT32,  33M)     |
+    |                               |   /dev/sda3 (SWAP,      SWAP,   >512MB)  |
+    |                               |   /dev/sda4 (ROOTFS,    EXT4,   *)       |
+    |                               |                                          |
+    |    GPT with UEFI Support      |   /dev/sda1 (BIOS BOOT, EXT4,   128MB)   |
+    |    wihout physical swap       |   /dev/sda2 (UEFI ESP,  FAT32,  33M)     |
+    |                               |   /dev/sda3 (ROOTFS,    EXT4,   *)       |
+    |                               |                                          |
+    +-------------------------------+------------------------------------------+
+    
+    * In most cases, the user will desire the root partition to span the remain 
+      space on the disk device.
+    * Building big packages like rust or firefox from source require a large 
+      amount of memory. Make sure that your physical RAM and swap add up to 
+      ~10GB or more of size. Or consider the usage of a dynamical growing swap 
+      file.
+
+      
+    [2.2] Swap Space
+    ____________________________________________________________________________
+    
+    The purpose of swap space is to provide disk storage to the kernel when 
+    internal memory (RAM) is under pressure. When the system is not going to run 
+    memory intensive applications or the system has lots of memory available, 
+    then it probably does not need much swap space. However, swap space is also 
+    used to store the entire memory in case of hibernation. If the system is 
+    going to need hibernation, then a bigger swap space is necessary, often at 
+    least the amount of memory installed in the system. 
+    
+    Note that a physical swap partition is not required as a swap file can be 
+    used, created/specified and resized later in the KISS Linux installation
+    process. 
```

Comment:
We should emphasis that swap files can be created in any number and at any time, not just during installation. I'm not sure if their creation belongs in [3.2] like it would with swap partitions (for which we should also build into an example), but we could at the very least link to [this](https://wiki.archlinux.org/index.php/Swap#Swap_file).

Reply:
what are the benefits/downsides to swap partition vs swap file?
if a swap file is as performant as swap partition, i think we can recommend a swap file, as resizing is much easier

Reply:
I think they're roughly identical performance-wise, assuming there is no fragmentation on the file.

Reply:
so there seems to be two benefits to having this guide use swap file
* more flexible than partitioning
* simplifies partitioning section by having swapfile be a different section

Reply:
Indeed. 
Swap partitions are probably better on spinning disks, because they can be put closer to the beginning of the disk to maximize speed. So they shouldn't be dropped entirely, but instead just emphasise the utility and ease of use swapfiles provide in our wonderful NVME future.

Reply:
A further thought: zswap and zram are both excellent alternatives to the historical swap partition.  Perhaps this should be a second article which this one can link to. 

I'll get to work on it.